presto-product-tests/README.md

```diff
@@ -143,6 +143,12 @@ presto-product-tests/bin/run_on_docker.sh <profile> -t com.facebook.presto.tests
 presto-product-tests/bin/run_on_docker.sh <profile> -t sql_tests.testcases.system.selectInformationSchemaTables
 ```
 
+Profile specific tests are tests which runs only for that particular profile. It confirms if the profile is same as what is claims.
```

Comment:
@petroav - Linking of docs
If user wants to have more information on Kerberos/Impersonation, how to configure, which properties to set, etc. The link to Hive connector docs will be useful for that.
I added Presto docs as a backup. In case, the link for Hive Connector changes.